lk2nd/smb1360/smb1360-qcom.c

```diff
@@ -16,6 +16,12 @@ static const struct smb1360_battery batteryB = {
 	.profile = SMB1360_BATTERY_PROFILE_B,
 };
 
+static int abs(unsigned int n) {
```

Comment:
This seems like exactly the opposite of what would be logical, wouldn't it make more sense to do

```suggestion
static unsigned int abs(int n) {
```

then you can also get rid of the weird cast below.